vpr/src/route/rr_graph.cpp

```diff
@@ -707,8 +711,8 @@ void create_rr_graph(const t_graph_type graph_type,
 
     print_rr_graph_stats();
 
-    //Write out rr graph file if needed
-    if (!det_routing_arch->write_rr_graph_filename.empty()) {
+    //Write out rr graph file if needed - Currently, writing the flat rr-graph is not supported since loading from a flat rr-graph is not supported
```

Comment:
We cannot read/write a flat RR graph. A non-flat RR graph can be read, and additional resources can be added to it. Inter-cluster part of the RR graph is built before placement. At that stage, the is_flat flag is false. When routing starts, the flag is true. So, even if the flat router is enabled fro the command line parameters, it will write the inter-cluster part.

Reply:
Should probably say this ... update the comment to be more explicit.